eras/byron/ledger/impl/cardano-ledger-byron.cabal

```diff
@@ -252,7 +252,7 @@ library
         quiet,
         resourcet,
         streaming,
-        streaming-binary >=0.2 && <0.3,
+        streaming-binary >=0.2 && <0.4,
```

Comment:
When updating bounds in cabal file, make sure to bump up the patch version of the package, otherwise it will not get released, so the new version should be: `1.0.0.3`

Reply:
Done!